c7n/resources/cw.py

```diff
@@ -1071,3 +1071,68 @@ class resource_type(TypeInfo):
     source_mapping = {
        "describe": DescribeWithResourceTags,
     }
+
+
+@resources.register("delivery-destination")
```

Comment:
rename to log-designation so it matches up to the other cloud watch log resources on prefix.